src/main/kotlin/com/lovelysystems/gradle/S3DownloadFile.kt

```diff
@@ -53,38 +53,16 @@ abstract class S3DownloadFile : DefaultTask() {
         val region = regionOverride.orNull ?: project.awsSettings.region
 
         val s3Client = S3AsyncClient.builder()
-            .credentialsProvider(ProfileCredentialsProvider.create(profile))
+            .credentialsProvider(
+                AwsCredentialsProviderChain.of(
+                    EnvironmentVariableCredentialsProvider.create(),
+                    ProfileCredentialsProvider.create(profile),
+                )
+            )
             .region(Region.of(region))
             .build()
 
-        HeadBucketRequest.builder().bucket(bucket.get()).build().let { req ->
```

Comment:
I originally already changed this in the previous pr but it got lost during some of the reverts.